### PR TITLE
Version 0.7

### PR DESCRIPTION
**Pull request by ErikBjare**
This is planned to be the last in the `0.x` series, will be followed by `1.0` if all goes according to plan.

- [ ] Update submodules
  - [ ] Merge current set of stable improvements
    - [x] https://github.com/ActivityWatch/aw-client/pull/12
    - [ ] https://github.com/ActivityWatch/aw-core/pull/23
- [ ] Packaging
  - [x] Unify build/install/CI scripts using makefile (see #43)
  - [x] Fix macOS packaging (see https://github.com/ActivityWatch/activitywatch/issues/27)
  - [ ] Create Debian package
- [ ] Passing builds in CI
  - [x] Travis
  - [x] Run all tests in submodules (~~use pytest for test discovery~~, call their `make test`)
  - [x] AppVeyor (might be delayed to a later version)
     - [ ] Upload artifact on releases

### CODE REVIEW

File: arch-package/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+epoch=
+pkgdesc="Log what you do on your computer. Simple, extensible, no third parties."
+arch=('i686' 'x86_64')
+url="https://github.com/ActivityWatch/activitywatch"
+license=('MPL2')
+groups=()
+depends=('python>=3.5.1')
+makedepends=('python>=3.5.1' 'python-setuptools')
+checkdepends=()
+optdepends=()
```
Comment (johan-bjareholt) [inline arch-package/PKGBUILD]: what's the point of all these empty fields?

Comment (ErikBjare) [inline arch-package/PKGBUILD]: Boilerplate I still might need. The PKGBUILD isn't ready and this is my first time packaging for Arch.

**Comment by ErikBjare** [comment]
Note-to-self: Merge in master so that Travis passes.

**Comment by johan-bjareholt** [comment]
That commit list, wow. Great work!

**Comment by ErikBjare** [comment]
I really want to merge this into master as soon as possible so we can get the new, nicer README on the main page of the repo.

**Comment by ErikBjare** [comment]
I'm going to do a WIP merge since the stuff in here is probably more stable than what's in master.

**Pull request merged**